### PR TITLE
Beret-B-Gone, an industrial level cleaner

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## About The Pull Request

Removes the excessive number of berets.
Makes the security beret, (sliiight armor), recolorable
## Why It's Good For The Game

We don't need this many berets when we have the variant that is player colorable.
## Changelog
:cl:
del: Removed all station berets excluding the normal, security, and HOS berets.
qol: Security Beret is now recolorable
/:cl:

### CODE REVIEW

**Comment by SimplySmiley123** [comment]
Don't tell me this removes the french beret
How will i make people french now

**Comment by MrMelbert** [comment]
I put my foot down at the Head of Security's beret, even though I'm a cap user

**Comment by mikederkan** [comment]
But... why?

**Comment by MrMelbert** [comment]
Also I said I'd block this unless it adds some form of hat to replace them, let people have hats damn it

File: code/modules/clothing/outfits/ert.dm [hunk 1]
```diff
@@ -473,8 +473,8 @@
 	if(!istype(mod))
 		return
 	var/obj/item/mod/module/hat_stabilizer/hat_holder = locate() in mod.modules
-	var/obj/item/clothing/head/helmet/space/beret/beret = new(hat_holder)
-	hat_holder.attached_hat = beret
+	var/obj/item/clothing/head/helmet/space/helm = new(hat_holder)
```
Comment (necromanceranne) [inline code/modules/clothing/outfits/ert.dm:476]: If I wasn't already convinced you're making a troll pr, this definitely would be a good hint.

**Comment by dwasint** [comment]
Some people may have said that swing combat is gonna cause a player exodus, some say its gonna be wallening. They are both wrong, this is the PR that will kill TG as we know it. When we needed MSO the most they vanished. Please return MSO and rightfully smite this PR where it stands.

**Comment by LemonInTheDark** [comment]
> Also I said I'd block this unless it adds some form of hat to replace them, let people have hats damn it

I can live with this, tho I don't totally get it. I dislike berets and how we've used them because they make something unique generalized.
We already have a general way to get a beret on station. I don't entirely understand the argument of "letting people have the hats" cause they already do, s in the wardrobe (might need to be tweaked mind, may have said something against that at some point idk old me is a mystery) but I realize that's more out of the way and.. underexposed I guess?

Having a placeholder job hat is reasonable if that's what you're suggesting, but I get the feeling placeholder is not realistic, given how this codebase works.

Removing em from erts is silly, don't do that.

**Comment by Addust** [comment]
make sure civilian crew can still acquire an unarmoured beret or something

**Comment by DATA-xPUNGED** [comment]
Removing it from science and medical is whatever, but you're removing the *hos beret*???

**Comment by Jackal-boop** [comment]
bro just let people have fashion damn

**Comment by atlasle** [comment]
good pr, merge it

**Comment by Bluedino1025** [comment]
coders when they see players enjoy something (this simply cannot be)

**Comment by bluetshirtguy** [comment]
Coders on their way to remove something other people enjoy.

**Comment by deathrobotpunch** [comment]
this is 100 precent a troll PR https://github.com/tgstation/tgstation/pull/78526#issuecomment-1732434292

**Comment by ZephyrTFA** [comment]
the fuck you doing git

**Comment by ZephyrTFA** [comment]
☄️

**Comment by Dawnseer** [comment]
I'd be rather sad if I lost my medical beret.

**Comment by DrDiasyl** [comment]
The PR says that it doesn't remove Security berets, yet the file changes show that it does.

File: _maps/map_files/CTF/downtown.dmm [hunk 2]
```diff
@@ -2126,9 +2126,6 @@
 /obj/structure/rack{
 	resistance_flags = 64
 	},
-/obj/item/clothing/head/beret/sec/navywarden{
```
Comment (NamelessFairy) [inline _maps/map_files/CTF/downtown.dmm:2129]: Could you replace this with a recolored blue beret rather than just leaving this empty?

**Comment by SansSarif** [comment]
> We don't need this many berets when we have the variant that is player colorable.

is there any intention of increasing the round start available number of berets if the explicit intention is for people to use the autodrobe berets exclusively

**Comment by ZephyrTFA** [comment]
> > We don't need this many berets when we have the variant that is player colorable.
> 
> is there any intention of increasing the round start available number of berets if the explicit intention is for people to use the autodrobe berets exclusively

sec berets yes, normal ones no. the vendor already has ten, if you need more buy a refill from cargo.

**Review by Ghommie — CHANGES_REQUESTED** [review]
As much as I can get behind squashing all types into one, this PR is also huge middle finger to anyone (but security) who wants a beret with the colors of their department/job, because now they'll have to walk to a ClothesMate and try to replicate that color scheme from memory or at a guess. Perhaps on Skyrat, with the loadout feature, it isn't much of an issue, but here it's got to be a nuisance doing it over and over.

EDIT: This was meant to be a general comment rather than a review, but since I'm at it, maybe you should add a generic badget beret to the autodrobe. The RD's beret was of that variant.

**Pull request closed without merging**